reform_cage.py

```diff
@@ -0,0 +1,176 @@
+"""
+From a two-component cage .mol file, returns:
+
+	1) The SMILES code of the building blocks.
+	1.1) The active coordination number of the linkers.
+        1.2) The active functional group.
+        1.3) The number of times a building block has been used in a cage.
+
+	2) The topology of the cage.
+
+        3) Optional Arguments.
+        3.1) Optional (-c): Reform the unrelaxed cage from .mol file.
+        3.2) Optional (-b): Reform the unrelaxed building-blocks (bb).
+
```

Comment:
You mix spaces and tabs here on the various lines here, it would be better to just use spaces. Most editors should do this change automatically for you.